var/spack/repos/builtin/packages/rocm-cmake/package.py

```diff
@@ -50,6 +51,7 @@ class RocmCmake(CMakePackage):
         "6.0.2",
         "6.1.0",
         "6.1.1",
+        "6.1.2",
```

Comment:
inside the rocm-cmake , can you remove the dead code -
def cache_test_sources(self):
if self.spec.satisfies("@:5.1.0"):
            return
def test(self):
        if self.spec.satisfies("@:5.1.0"):
            print("Skipping: stand-alone tests")
            return

Reply:
Removing deprecated version checks